pupil_src/launchables/eye.py

```diff
@@ -221,17 +221,24 @@ def get_timestamp():
         default_plugins = [
             # TODO: extend with plugins
             (default_capture_name, default_capture_settings),
-            ("UVC_Manager", {}),
-            # Detectors needs to be loaded first to set `g_pool.pupil_detector`
-            (default_2d.__name__, {}),
-            (default_3d.__name__, {}),
-            ("NDSI_Manager", {}),
-            ("HMD_Streaming_Manager", {}),
-            ("File_Manager", {}),
-            ("PupilDetectorManager", {}),
-            ("Roi", {}),
         ]
 
+        # Detectors needs to be loaded first to set `g_pool.pupil_detector`
+        for pupil_detector in [default_2d, default_3d]:
+            if pupil_detector is not None:
+                default_plugins.append((pupil_detector.__name__, {}))
```

Comment:
Do we actually need `default_2d`/`default_3d` now?
Maybe just a list `default_detectors`, which includes 2d and if available pye3d?
Then we don't have this problem of setting one to `None` and checking for it.
Not sure where else we reference the `default_2d`/`default_3d` from pupil_detector_plugins/__init__.py?